cmd/thanos/compact.go

```diff
@@ -183,6 +183,12 @@ func runCompact(
 		return errors.Wrap(err, "create compactor")
 	}
 
+	readinessProber, err := metricHTTPListenGroup(g, logger, reg, httpBindAddr, component)
+	if err != nil {
+		readinessProber.SetNotHealthy(err)
```

Comment:
IMO we don't need this here, because for all the other errors we don't set it. And this doesn't really do anything no?
For example 4 lines above:
```
comp, err := tsdb.NewLeveledCompactor(reg, logger, levels, downsample.NewPool())
	if err != nil {
		return errors.Wrap(err, "create compactor")
	}
```

Reply:
You are right.. I'll get rid of it. Thanks fro poinitng it out :+1: